lib/float/ne.sail

```diff
@@ -42,8 +43,7 @@ function float_is_ne ((op_0, op_1)) = {
                 else fp_eflag_none;
 
   let is_nan  = float_is_nan (op_0) | float_is_nan (op_1);
-  let is_zero = float_is_zero (op_0) & float_is_zero (op_1);
-  let is_ne   = not (is_nan) & (op_0 != op_1) & not (is_zero);
+  let is_ne   = not (is_nan) & float_is_ne_internal ((op_0, op_1));
```

Comment:
Maybe this is simpler? Either way is probably fine, but this reduces the number of internal arithmetic functions since they are doing basically the same thing.
 
`let is_ne   = not (is_nan) & not (float_is_eq_internal ((op_0, op_1)));`

Reply:
I think either way is fine. Was just a suggestion to reduce the amount of code. If you feel strongly about keeping it as separate functions, that works for me.

Reply:
Great! Thanks for your suggestion. Actually sometimes I am also entangled which one is better, and finally choose the simpler one from reading.

About the code use, I think we can invoke float_is_eq_internal from float_is_ne_internal directly, let me update it.